gateway/src/main/resources/application.yml

```diff
@@ -0,0 +1,173 @@
+#        provider:
+#          uaa:
+#            authorization-uri: http://localhost:8090/uaa/oauth/authorize
+#            token-uri: http://localhost:8090/uaa/oauth/token
+#            user-info-uri: http://localhost:8090/uaa/userinfo
+#            user-name-attribute: sub
+#            jwk-set-uri: http://localhost:8090/uaa/token_keys
+  cloud:
+    gateway:
+      enabled: true 
+      global-filter.websocket-routing.enabled: true
+      metrics.enabled: true
+      default-filters:
+      - SecureHeaders
+      - TokenRelay
+      - RemoveSecurityHeaders
+      # AddSecHeaders appends sec-* headers to proxied requests based on the
+      # georchestra.gateway.default-headers and georchestra.gateway.servies.<service>.headers config properties 
+      - AddSecHeaders
+      filter:
+        secure-headers:
+          enabled: true
+          disable:
+          - content-security-policy
+          frame-options: SAMEORIGIN
+          xss-protection-header: 0
+          referrer-policy: no-referrer
+          #content-security-policy: script-src 'self' 'unsafe-eval'; object-src 'self';
+      routes:
+      - id: root
+        uri: ${georchestra.gateway.services.header.target}
+        predicates:
+        - Path=/
+        filters:
+        - RewritePath=/, /header
+      - id: header
+        uri: ${georchestra.gateway.services.header.target}
+        predicates:
+        - Path=/header,/header/**
+    
+      - id: analytics
+        uri: ${georchestra.gateway.services.analytics.target}
+        predicates:
+        - Path=/analytics/**
+    
+      - id: atlas
+        uri: ${georchestra.gateway.services.atlas.target}
+        predicates:
+        - Path=/atlas/**
+    
+      - id: console
+        uri: ${georchestra.gateway.services.console.target}
+        predicates:
+        - Path=/console/**
+    
+      - id: extractorapp
+        uri: ${georchestra.gateway.services.extractorapp.target}
+        predicates:
+        - Path=/extractorapp/**
+    
+      - id: geonetwork
+        uri: ${georchestra.gateway.services.geonetwork.target}
+        predicates:
+        - Path=/geonetwork/**
+    
+      - id: geoserver
+        uri: ${georchestra.gateway.services.geoserver.target}
+        predicates:
+        - Path=/geoserver/**
+
+      - id: mapfishapp
+        uri: ${georchestra.gateway.services.mapfishapp.target}
+        predicates:
+        - Path=/mapfishapp/**
+
+      - id: geowebcache
+        uri: ${georchestra.gateway.services.geowebcache.target}
+        predicates:
+        - Path=/geowebcache/**
+
+      - id: mapstore
+        uri: ${georchestra.gateway.services.mapstore.target}
+        predicates:
+        - Path=/mapstore/**
+
+      - id: datafeeder
+        uri: ${georchestra.gateway.services.datafeeder.target}
+        predicates:
+        - Path=/datafeeder/**
+
+      - id: import
+        uri: ${georchestra.gateway.services.import.target}
+        predicates:
+        - Path=/import/**
+
```

Comment:
would it be relevant to add some properties here to define the access, like:

```yaml
  allow: ROLE_ADMINISTRATOR
  deny: ALL
```

(see previous comment about `security-mappings.xml` file)